src/mongocxx/write_concern.hpp

```diff
@@ -52,8 +52,6 @@ class uri;
 ///
 /// @see http://docs.mongodb.org/manual/core/write-concern/
 ///
-/// @todo this interface is terrible -- it's exactly what you get with the c-driver essentially.
```

Comment:
I don't think there is anything particularly wrong with it. Who wrote the comment?